src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/Browsing/OfflineMapsView.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<Page
+    x:Class="OfflineWorkflowSample.Views.OfflineMapsView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required"
+    Padding="5"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <GridView ItemsSource="{x:Bind ViewModel.LocalContentViewModel.Items,Mode=OneWay}"
```

Comment:
Maps that I take offline do not show up in this view until I restart the app. Seems like a missing update/event somewhere.